hands-on-exercise/5-ibc-adv/6-ibc-app-steps.md

```diff
@@ -103,7 +103,7 @@ The required steps to implement can be found in the [ibc-go docs](https://ibc.co
     * Packet callbacks
 * Bind to a port(s).
 * Add keeper methods.
-* Define your packet data and acknowledgment structs as well as how to encode/decode them.
+* Define your packet data and acknowledgement structs as well as how to encode/decode them.
```

Comment:
@AndrewB9lab  😬 I thought with an "e" was used both sides.

Reply:
Crossed lines on this, I thought you were correcting to avoid the inconsistency with the code. But ack-without-E. I'll fix this now, and check to make sure I don't touch anything with "code context".